microsoft-edge/devtools-guide-chromium/performance/performance-tool-overview.md

```diff
@@ -0,0 +1,410 @@
+
+1. Right-click the demo webpage, and then select **Inspect**.
+
+   DevTools opens.
+
+1. In DevTools, open the **Performance** tool:<!-- end of added para's -->
+
+   ![The Performance tool, initial view](./performance-tool-overview-images/performance-tool-inprivate.png)
+
+1. Click the **Capture settings** (![Capture settings](./performance-tool-overview-images/capture-settings-icon.png)) button.
+
+   DevTools displays settings for capturing performance metrics.
+
+1. In the **CPU** drop-down list, select **4x slowdown - recommended**:
+
+   ![CPU throttle](./performance-tool-overview-images/capture-settings.png)
+
+   DevTools throttles your CPU so that it's 4 times slower than usual.  A warning icon is displayed on the **Performance** tool's tab, to remind you that throttling is enabled.
+
+   If you want to ensure that pages work well on low-end mobile devices, set **CPU** to **6x slowdown**.
+
+
+<!-- ---------- -->
+###### Configure your environment to better match that of your users
+
+With the field data fetch set up as described in [Compare your experience to the experience of your users](#compare-your-experience-to-the-experience-of-your-users) above, the **Performance** tool provides you with recommendations on how to configure your environment to better match the experience of your users.
+
+To configure your environment:
+
+1. In the **Performance** tool, in each metric card, expand the **Consider your local test conditions** section, if any, and read the recommendations.
+
+   <!-- ![The 'Consider your local test conditions' sections expanded in each metric card](./performance-tool-overview-images/env-recs.png) -->
+
+   <!-- Looks like in this example, to better match the experience of your users, you might want to use a common desktop screen size and throttle down the CPU and network. -->
+
+1. Apply the recommendations.  For example, set your viewport to one of the common screen sizes (for example, 720p or 1080p).  To emulate specific devices and screen sizes, you can use device emulation in the **Elements** tool; see [Emulate mobile devices (Device Emulation)](../device-mode/index.md).
+
+1. Some percentage of users of the website use desktops to browse.  To make sure that you compare your local metric scores to the correct field data, you can select **Desktop** from the **Field data** > **Device** drop-down list.
+
+1. In the **Environment settings** section, set the **Network** drop-down list to, for example, **Fast 4G**, and **CPU** to, for example, **20x slowdown**.  You may also make sure to select the **Disable network cache** checkbox in the same section.
+
+1. Reload the webpage, interact with the webpage to capture your local INP, and compare the metric scores again.
+
+   ![The environment is configured to match the real-world user experience](./performance-tool-overview-images/env-config.png)<!-- todo: redo w/ edge -->
```

Comment:
redo png w/ Edge - The environment is configured to match the real-world user experience - env-config.png

Reply:
upstream filename performance/overview.md
This Outdated comment is in /performance/performance-tool-overview.md
Latest filename in PR is instead like upstream: /performance/overview.md